db/models.py

```diff
@@ -0,0 +1,92 @@
+from sqlalchemy import Column, Integer, String, Float, ForeignKey, Table, Boolean
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import relationship
+
+Base = declarative_base()
+
+
+class Client(Base):
+    __tablename__ = 'client'
+    id = Column(Integer, primary_key=True)
+    name = Column(String(50))  # Assuming a name field to identify the client
+    locations = relationship('Location', back_populates='client')
+
+
+class Location(Base):
+    __tablename__ = 'location'
+    id = Column(Integer, primary_key=True)
+    latitude = Column(Float)
+    longitude = Column(Float)
+    marked = Column(Boolean)
```

Comment:
What do you plan on using marked for?

Reply:
It was on the db schema in the definition of the project. I am not sure of the purpose for now.